model_selection/imbalanced/imbalanced_metrics.ipynb

```diff
@@ -1300,7 +1300,16 @@
     "\n",
     "Our true average rate of travel, automagically adjusted for time spent traveling in each direction = 15 mph! In this example, using harmonic mean helps us find relationship with datasets of rates/ratios/fractions over different length/periods. By using reciprocals, it already accounts for the proportion of time that was implicit in the rates.\n",
     "\n",
-    "With this knowledge in mind, let's turn our heads back to F1 score's calculation. Precision and recall both have true positives in the numerator, but they have different denominators. In order for an average to be valid, we need the values to be in the same scaled units. In our traveling distance example miles per hour need to be compared over the same number of hours. Thus, to take the average of precision and recall really only makes sense to average their reciprocals (adjusts for scale), thus the harmonic mean."
+    "With this knowledge in mind, let's turn our heads back to F1 score's calculation. Precision and recall both have true positives in the numerator, but they have different denominators. In order for an average to be valid, we need the values to be in the same scaled units. In our traveling distance example miles per hour need to be compared over the same number of hours. Thus, to take the average of precision and recall really only makes sense to average their reciprocals (adjusts for scale), thus the harmonic mean.\n",
+    "\n",
+    "If the business case warrants it, you may want to put more weight on recall than precision or vice-versa. In that case a more general version of the F score called F beta score could be useful. \n",
```

Comment:
More of a nitpick. Perhaps change the "you" to "we". e.g.

- we may want to put more weight.
- We focus more on recall. etc.

Reply:
You are right, I'll change that.

Reply:
Done @ethen8181